File: 75. Sort Colors/75. Sort Colors.py
class Solution(object):
    def sortColors(self, nums):
        low=0
        mid=0
        high=len(nums)-1
        while mid<=high:
            if nums[mid]==0:
                nums[low],nums[mid]=nums[mid],nums[low]
                mid+=1
                low+=1
            elif nums[mid]==1:
                mid+=1
            elif nums[mid]==2:
                nums[mid],nums[high]=nums[high],nums[mid]
                high-=1
    
        

File: 75. Sort Colors/75. Sort Colors.java
class Solution {
    public void sortColors(int[] nums) {
        
        int low=0,temp;
        int mid=0;
        int high=nums.length-1;
        
        while(mid<=high)
        {
            switch(nums[mid])
            {
                case 0:
                    temp=nums[mid];
                    nums[mid]=nums[low];
                    nums[low]=temp;
                    low++;
                    mid++;
                    break;
                case 1:
                    mid++;
                    break;
                case 2:
                    temp=nums[mid];
                    nums[mid]=nums[high];
                    nums[high]=temp;
                    high--;
                    break;
            }
        }
    }
}